tests/test_for_suite/test_for_cli/test_for_runner.py

```diff
@@ -52,11 +53,41 @@ def test_run_output(runner, model_file):
 
 
 @pytest.mark.skip(reason="TODO: Need to provide input somehow.")
-def test_run_output(runner, tmpdir):
+def test_run_output_TODO(runner, tmpdir):
```

Comment:
Is this TODO for a future PR?

Reply:
This test was left over from before. I merely added TODO to the test here so that my pycodestyle doesn't complain about the c/p `test_run_output` function name anymore.